marklogic-data-hub-central/ui/src/components/column-selector/column-selector.test.tsx

```diff
@@ -27,19 +27,32 @@ describe('Column selector component', () => {
   });
 
   test('Verify cancel button closes popover', () => {
-    const { getByText } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns}/>);
+    const { getByText } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns} primaryKey={''}/>);
     const cancelButton = getByText('Cancel');
     cancelButton.onclick = jest.fn();
     fireEvent.click(cancelButton);
     expect(cancelButton.onclick).toHaveBeenCalledTimes(1);
   });
 
   test('Verify apply button closes popover', () => {
-    const { getByText } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns}/>);
+    const { getByText } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns} primaryKey={''}/>);
     const applyButton = getByText('Apply');
     applyButton.onclick = jest.fn();
     fireEvent.click(applyButton);
     expect(applyButton.onclick).toHaveBeenCalledTimes(1);
   });
+
+  test('Verify primaryKey tooltip renders', async () => {
+    const { getByTestId } = render(<ColumnSelector popoverVisibility={true} setPopoverVisibility={jest.fn()} entityPropertyDefinitions={entityPropertyDefinitions} selectedPropertyDefinitions={selectedPropertyDefinitions} setColumnSelectorTouched={jest.fn()} columns={columns} primaryKey={'customerId'}/>);
+    expect(getByTestId('pk-tooltip')).toBeInTheDocument();
+  });
+
+
+  test('Verify column selector tooltips renders', async () => {
```

Comment:
there is already a test for column selector visibility and the test to verify tooltip should go in there. All tests should be grouped together when possible. We should try to avoid one line tests. A typical user would not login to HC and navigate to explorer tile to verify tooltip or popover closes.

Lets remember the Guiding Principle of RTL whenever tests are added -- "The more your tests resemble the way your software is used, the more confidence they can give you."

Reply:
Merged tests.